File: BLL/Models/RegistrationModel.cs
namespace BLL.Models
{
    public class RegistrationModel
    {
        public string Email { get; set; }
        public string Password { get; set; }
        public string PasswordRepeat { get; set; }

        public string RoleName { get; set; }
        public string Nickname { get; set; }
    }
}

File: ForumFrontend/src/app/models/registration.model.ts
export class Registration {
    Email: string;
    Password: string;
    PasswordRepeat: string;
    RoleName: string;
    Nickname: string;
    constructor(email: string, password: string, password2: string, rolename: string, nickname: string){
        this.Email = email;
        this.Password = password;
        this.PasswordRepeat = password2;
        this.RoleName = rolename;
        this.Nickname = nickname;
    }
}
